### PR TITLE
Variant and FAQ pages typos

### PR DESCRIPTION
**Pull request by randompearl-stack**

### CODE REVIEW

File: static/docs/xiangqi.md [hunk 1]
```diff
@@ -143,21 +143,21 @@ Sometimes, some players would use a "." instead of "=".
 
 ### Disambiguation
 
-If two of the same piece are in the same file, then a + or - to specify if it's the more advanced piece (+) or the piece that's at a lower rank (-). For example, if two chariots are on the same file but on opposite sides of the river, and you wanted to move to the farther one to the center, that would be R+=5.
+If two of the same pieces are in the same file, then a + or - to specify if it's the more advanced piece (+) or the piece that's at a lower rank (-). For example, if two chariots are on the same file but on opposite sides of the river, and you wanted to move to the farther one to the center, that would be R+=5.
```
Comment (Fulmene) [inline static/docs/xiangqi.md:146]: I don't think simply changing it to "pieces" is correct here. This sentence is about having two pieces of the same type **on** (yeah this should be changed too) the same file.

File: static/docs/xiangqi.md [hunk 2]
```diff
@@ -181,13 +181,13 @@ E | 2.5
...
 
-The following information is courtesy of [this site](http://www.shakki.info/english/openings.html)
+The following information is courtesy of [Shakki.info](http://www.shakki.info/english/openings.html).
 
 The most common opening move is the central cannon, which is a pretty obvious move because it opens aggression down the central file. About 70% of games start this way, so it's probably the best way to start learning the game.
 
 ![Cannon opening](https://github.com/gbtami/pychess-variants/blob/master/static/images/XiangqiGuide/CannonOpening.png)
 
-There are four very popular defenses, and a fifth will also be mentioned.
+There are four very popular defenses and a fifth will also be mentioned.
```
Comment (Fulmene) [inline static/docs/xiangqi.md:190]: I think keeping the comma is more natural.

File: static/docs/capablanca.md [hunk 3]
```diff
@@ -26,27 +26,27 @@ The piece is often symbolized with a combination of a knight and bishop; most va
 
 The archbishop is unique in that it is the only piece that can checkmate on its own, which you may be able to appreciate if you look at its movement/attack pattern.
 
-The value of an archbishop is considered slightly better than a rook, but less than the chancellor and queen.
+The value of an archbishop is considered slightly better than a rook, but less than the chancellor or queen.
```
Comment (Fulmene) [inline static/docs/capablanca.md:29]: I think "and" is already correct. The value of the archbishop is less than the chancellor and the queen.

File: static/docs/capablanca.md [hunk 4]
```diff
@@ -8,15 +8,15 @@ Capablanca chess (or Capablanca's chess) was created by World Chess Champion Jos
 
 The game is played on a 10 x 8 board, with additional files for the new Archbishop (Knight/Bishop) and Chancellor (Knight/Rook) pieces between the Bishop and Knight. The archbishop is on the queen side, and the chancellor is on the king side. For castling, the king then moves three squares instead of two. Pawns may promote to the archbishop and chancellor as well.
 
-There are several variants that have different starting positions. On this site, Gothic chess is one such variant available, where the  starting positions are swapped.
+There are several variants that have different starting positions. On this site, Capablanca960 is one such variant available, where the  starting positions are randomized.
```
Comment (Fulmene) [inline static/docs/capablanca.md:11]: This place should mention the alternate starting position options in the Create a Game menu, not 960.

File: static/docs/capablanca960.md [hunk 5]
```diff
@@ -32,18 +33,18 @@ The piece is often symbolized with a combination of a knight and bishop; most va
 
 The archbishop is unique in that it is the only piece that can checkmate on its own, which you may be able to appreciate if you look at its movement/attack pattern.
 
-The value of an archbishop is considered slightly better than a rook, but less than the chancellor and queen.
+The value of an archbishop is considered slightly better than a rook, but less than the chancellor or queen.
```
Comment (Fulmene) [inline static/docs/capablanca960.md:36]: Same as the Capablanca file.

File: static/docs/capahouse.md [hunk 6]
```diff
@@ -30,22 +30,22 @@ The piece is often symbolized with a combination of a knight and bishop; most va
 
 The archbishop is unique in that it is the only piece that can checkmate on its own, which you may be able to appreciate if you look at its movement/attack pattern.
 
-The value of an archbishop is considered slightly better than a rook, but less than the chancellor and queen.
+The value of an archbishop is considered slightly better than a rook, but less than the chancellor or queen.
```
Comment (Fulmene) [inline static/docs/capahouse.md:33]: Same as the Capablanca file.

File: static/docs/capahouse960.md [hunk 7]
```diff
@@ -38,21 +39,21 @@ The piece is often symbolized with a combination of a knight and bishop; most va
 
 The archbishop is unique in that it is the only piece that can checkmate on its own, which you may be able to appreciate if you look at its movement/attack pattern.
 
-The value of an archbishop is considered slightly better than a rook, but less than the chancellor and queen.
+The value of an archbishop is considered slightly better than a rook, but less than the chancellor or queen.
```
Comment (Fulmene) [inline static/docs/capahouse960.md:42]: Same as the Capablanca file.

File: static/docs/chak.md [hunk 8]
```diff
@@ -55,14 +55,14 @@ Other game-end conditions result in a draw:
...
+The Divine King moves up to *two* squares in any direction. It can also pass through squares that are threatened by the opponent’s pieces (in other words, it is not like the Western Chess king that cannot castle through check). Remember that as a promoted piece, it cannot go back beyond the river! The divine king has the ability to end the game by reaching the altar without moving into check. Because of its range, the divine king has the ability to deliver checkmate to an unpromoted king.
 
-Note that the symbol used only differs slightly from the king’s. They are meant to be the same, but the slight difference is to help players see the promotion and also for the two types of king to be distinct on the board editor.
+Note that the symbol used only differs slightly from the king’s. They are meant to be the same, but the slight difference is to help players see the promotion and also for the two types of kings to be distinct on the board editor.
```
Comment (Fulmene) [inline static/docs/chak.md:65]: This doesn't sound correct to me. I know for sure the king word should be singular. There is only a single king piece on each side, but he can promote to a different type.

Maybe "two **forms** of **the king**"

File: static/docs/chak.md [hunk 9]
```diff
@@ -168,21 +168,21 @@ where green is in zugzwang; moving the king to either side allows white to respo
...
+All these examples are bareboned in a sense that if either side decided to move any of their pieces away from around the temple, they can no longer form a fortress. If both sides have a bareboned fortress like in the above diagrams, the game is a dead draw. On the other hand, if one side has a bareboned fortress and the other has a fortress plus any extra piece, as long as that extra piece can safely cross the river without being captured and join the attack on the fortress, it's almost always a win for the side with the extra piece.
 
 ### Other defenses
 
 #### Jaguar
 
-In a lone Jaguar vs a Divine King endgame either side can force a draw by repetition
+In a lone Jaguar vs a Divine King endgame, either side can force a draw by repetition...
```
Comment (Fulmene) [inline static/docs/chak.md:185]: The three dots here look weird. Is there a reason for it?

File: static/docs/gorogoroplus.md [hunk 10]
```diff
@@ -11,4 +11,4 @@ Gorogoro+ is essentially shogi on a 5x6 board.
 
 As this game is based off shogi, please see the corresponding guide for basic shogi rules first if you are not familiar with them.
 
-The starting setup is as above. Unlike standard shogi, there is no bishop or rook. There are only kings, pawns, lances, knights and generals. You have knight and lance in hand at the beginning of the game. Promotion is only done in the last two ranks. 
+The starting setup is as above. Unlike standard shogi, there is no bishop or rook. There are only kings, pawns, lances, knights and generals. You have a knight and lance in hand at the beginning of the game. Promotion is only done in the last two ranks.
```
Comment (Fulmene) [inline static/docs/gorogoroplus.md:14]: Should be "a knight and **a** lance".

File: static/docs/orda.md [hunk 11]
```diff
@@ -41,15 +41,15 @@ A yurt is a mobile home for Mongol and Turkic peoples in the steppes of Asia. Th
...
 The kheshigs were the elite imperial guard for the Mongol royalty. Appropriately, it is incredibly difficult for the Kingdom to checkmate the khan without at least eliminating one of his kheshigs first.
 
 ### Horse Archer (A)
 
 ![Horse Archer](https://github.com/gbtami/pychess-variants/blob/master/static/images/CVariantsGuide/Archer.png)
  
 The Horse Archer, or simply abbreviated Archer, is a unique divergent piece that moves and attacks differently. The archer moves as a knight but captures as a bishop. Because the archer is not colorbound, its value is greater than its bishop counterpart.
-Horse Archers were one of the two core components of the Mongol cavalry, functioning as the light cavalry. Their speed and prowess as mounted archers made them a unique threat. Their ability to quickly position themselves for a deadly skewer or fork make them a dangerous threat for the Kingdom.
+Horse Archers were one of the two core components of the Mongol cavalry, functioning as the light cavalry. Their speed and prowess as mounted archers made them a unique threat. Their ability to quickly position themselves for a deadly skewer or fork makes them a dangerous threat to the Kingdom.
```
Comment (Fulmene) [inline static/docs/orda.md:52]: I think "**for** the kingdom" sounds more natural here.

File: static/docs/seirawan.md [hunk 12]
```diff
@@ -1,12 +1,12 @@
...
 
 ## Rules
 
 Unlike the above variants, this is played on a standard 8 x 8 board. The **elephant** is the rook/knight hybrid, while the **hawk** is the bishop/knight hybrid. Both of these pieces instead start off the board, although they are not to be dropped as in Crazyhouse.
 
-Instead, when a piece in a player’s first rank moves for the first time, these pieces may optionally come into the square evacuated by that piece. If all first rank pieces have moved or been captured, then any remaining elephant or hawk cannot be entered into play. For castling, the extra pieces can enter either of the original king or rook squares. The king cannot be in check during this process, similar to castling.
+Instead, when a piece in a player’s first rank moves for the first time, these pieces may optionally come into the square evacuated by that piece. If all first rank pieces have moved or been captured, then any remaining elephant or hawk cannot be entered into play. For castling, the extra pieces can enter either the original king or rook squares. The king cannot be in check during this process, similar to castling.
```
Comment (Fulmene) [inline static/docs/seirawan.md:9]: "either of" sounds more natural.

File: static/docs/seirawan.md [hunk 13]
```diff
@@ -20,21 +20,21 @@ The hawk (H) is a compound piece combining the moves of the **bishop** and **kni
 
 The hawk is unique in that it is the only piece that can checkmate on its own, which you may be able to appreciate if you look at its movement/attack pattern.
 
-The value of a hawk is considered slightly better than a rook, but less than the elephant and queen.
+The value of a hawk is considered slightly better than a rook, but less than the elephant or queen.
```
Comment (Fulmene) [inline static/docs/seirawan.md:23]: Same as the Capablanca file.

File: static/docs/shinobiplus.md [hunk 14]
```diff
@@ -4,7 +4,7 @@
...
 
-The game was initially considered balance by engine selfplay. However, player experience and also deeper NNUE engine skill revealed that the pink army had quite a decent advantage. Therefore, "Shinobi+" was created as a more balanced version. Most pieces were moved to being in the hand, and the new Fox piece was introduced. Additionally, black was allowed an earlier promotion rank.
+The game was initially considered balanced by engine selfplay. However, player experience and also deeper NNUE engine skill revealed that the pink army had quite a decent advantage. Therefore, "Shinobi+" was created as a more balanced version. Most pieces were moved to being in the hand, and the new Fox piece was introduced. Additionally, black was allowed an earlier promotion rank.
```
Comment (Fulmene) [inline static/docs/shinobiplus.md:7]: I think "also" should also be cut. "player experience and deeper NNUE engine skill revealed ..."

File: static/docs/shouse.md [hunk 15]
```diff
@@ -26,16 +26,16 @@ The hawk (H) is a compound piece combining the moves of the **bishop** and **kni
 
 The hawk is unique in that it is the only piece that can checkmate on its own, which you may be able to appreciate if you look at its movement/attack pattern.
 
-The value of a hawk is considered slightly better than a rook, but less than the elephant and queen.
+The value of a hawk is considered slightly better than a rook, but less than the elephant or queen.
```
Comment (Fulmene) [inline static/docs/shouse.md:29]: Same as the Capablanca file.

File: static/docs/capahouse.md [hunk 16]
```diff
@@ -30,22 +30,22 @@ The piece is often symbolized with a combination of a knight and bishop; most va
...
 
-The value of a chancellor is considered better than an archbishop, but equivalent or slightly less than a queen.
+The value of a chancellor is considered better than an archbishop, but equivalent to or slightly less than a queen.
 
 ## Strategy
 
-As in standard crazyhouse, the piece values don't align with the chess piece values, and as in standard crazyhouse, sacrificing material for fast development, attacks on the enemy King, or just for defence, are often wise. Sometimes it is better to reinforce defenders around one's King rather than try to save them. Here we may refer you to sources in standard crazyhouse strategy...
+As in standard crazyhouse, the piece values don't align with the chess piece values, and as in standard crazyhouse, sacrificing material for fast development, attacks on the enemy King, or just for defence are often wise. Sometimes it is better to reinforce defenders around one's King rather than try to save them. Here we may refer you to sources in standard crazyhouse strategy...
```
Comment (Fulmene) [inline static/docs/capahouse.md:51]: Keeping the comma is already correct. However, the following phrase should be "**is** often wise".

File: static/docs/shouse.md [hunk 17]
```diff
@@ -26,16 +26,16 @@ The hawk (H) is a compound piece combining the moves of the **bishop** and **kni
...
 
-The value of an elephant is considered better than a hawk, but equivalent or slightly less than a queen.
+The value of an elephant is considered better than a hawk, but equivalent to or slightly less than a queen.
 
 ## Strategy
 
-As in standard crazyhouse, the piece values don't align with the chess piece values, and as in standard crazyhouse, sacrificing material for fast development, attacks on the enemy King, or just for defence, are often wise. Sometimes it is better to reinforce defenders around one's King rather than try to save them. Here we may refer you to sources in standard crazyhouse strategy...
+As in standard crazyhouse, the piece values don't align with the chess piece values, and as in standard crazyhouse, sacrificing material for fast development, attacks on the enemy King, or just for defence are often wise. Sometimes it is better to reinforce defenders around one's King rather than try to save them. Here we may refer you to sources in standard crazyhouse strategy...
```
Comment (Fulmene) [inline static/docs/shouse.md:41]: Same as the Capahouse file.

**Review by Fulmene — CHANGES_REQUESTED** [review]

File: static/docs/spartan.md [hunk 18]
```diff
@@ -59,13 +59,13 @@ The warlord (*polemarchos*) moves as a bishop combined with a knight (i.e. archb
 
 ![Captain](https://github.com/gbtami/pychess-variants/blob/master/static/images/CVariantsGuide/SpartanCaptain.png)
  
-The captain (*tyntagmatarchos*) moves one or two squares in any direction orthogonally. It can jump over blocking friendly/enemy pieces to reach the second square.
+The captain (*tyntagmatarchos*) moves one or two squares in any direction orthogonally. It can jump over blocking friendly/enemy pieces from reaching the second square.
```
[the remark this answers is not in the document]
Comment (randompearl-stack) [reply static/docs/spartan.md:62]: my problem I misread

File: static/docs/chak.md (hunk 9 above)
Comment (randompearl-stack) [reply static/docs/chak.md:185]: The content after is 

> JaguarPerpetual

...except when the side with the Jaguar can just ignore their Jaguar being under attack and run their King towards the altar like in this position.

File: static/docs/orda.md (hunk 11 above)
Comment (randompearl-stack) [reply static/docs/orda.md:52]: doesn't "threat for" sound like its a threat that can be used by the Kingdom to gain an advantage?

Comment (Fulmene) [reply static/docs/orda.md:52]: "X is a threat for Y" just means "X threatens Y".

File: static/docs/chak.md (hunk 9 above)
Comment (Fulmene) [reply static/docs/chak.md:185]: That's fair.

File: static/docs/xiangqi.md (hunk 1 above)
Comment (Fulmene) [reply static/docs/xiangqi.md]: Should be "If two pieces of the same type ..."

**Review by Fulmene — APPROVED** [review]

**Pull request merged**